src/config/mockMode.js

```
@@ -143,9 +181,64 @@ class MockModeManager {
+                  select: (columns = '*') => {
+                    // Create a proper chainable select that supports .single()
+                    const selectChain = {
+                      single: () => Promise.resolve({
+                        data: existing,
+                        error: null
+                      })
+                    };
+                    
+                    // Make this object act like a Promise while preserving chainable methods
+                    return Object.assign(selectChain, {
+                      then: (onFulfilled, onRejected) => {
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)